tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/BaseRequestBuilderTests.cs

```diff
@@ -32,5 +37,107 @@ public void AppendSegmentToRequestUrl()
+            var parameter_second_function = "test-value";
+            var queryOptions = new List<Option>() { new QueryOption("filter", "name")};
+
+            // Create the composed function request builders
+            var composableFunctionRequestBuilder0 = new ComposableFunctionRequestBuilder0(requestUrl, client, parameter_first_function);
+            var composedRequestUrl = composableFunctionRequestBuilder0.RequestBuilder1(parameter_second_function).Request(queryOptions).RequestUrl;
+
+            var expected = @"https://localhost:443/microsoft.graph.composablefunction0(address='A1:B1')/microsoft.graph.composablefunction1(anotherValue='test-value')";
+
+            Assert.Equal(expected, composedRequestUrl);
+        }
+
+        /// <summary>
+        /// Test that null values are accepted for nullable parameters.
+        /// </summary>
+        [Fact]
+        public void ComposableFunctionWithNullParamTest()
+        {
+            var requestUrl = "https://localhost:443/microsoft.graph.composablefunction0";
+            var client = new BaseClient(" ", new HttpClient()); // base url needs to be a non-zero length string.
+            string parameter_first_function = null;
+            var parameter_second_function = "test-value";
+
+            // Create the composed function request builders
+            var composableFunctionRequestBuilder0 = new ComposableFunctionRequestBuilder0(requestUrl, client, parameter_first_function);
+            var composedRequestUrl = composableFunctionRequestBuilder0.RequestBuilder1(parameter_second_function).Request().RequestUrl;
+
+            var expected = @"https://localhost:443/microsoft.graph.composablefunction0(address=null)/microsoft.graph.composablefunction1(anotherValue='test-value')";
+
+            Assert.Equal(expected, composedRequestUrl);
+        }
+
+        /// <summary>
+        /// Test that parenthesis are added for a function without a parameter.
+        /// This will help distinguish functions without parameters from cast
+        /// syntax.
+        /// </summary>
+        [Fact]
+        public void ComposableFunctionWithNoParamsTest()
+        {
+            var requestUrl = "https://localhost:443/microsoft.graph.composablefunction0";
+            var client = new BaseClient(" ", new HttpClient()); // base url needs to be a non-zero length string.
+            var parameter_second_function = "test-value";
+
+            // Create the composed function request builders
+            var composableFunctionRequestBuilder0 = new ComposableFunctionRequestBuilder0(requestUrl, client);
+            var composedRequestUrl = composableFunctionRequestBuilder0.RequestBuilder1(parameter_second_function).Request().RequestUrl;
+
+            var expected = @"https://localhost:443/microsoft.graph.composablefunction0()/microsoft.graph.composablefunction1(anotherValue='test-value')";
+
+            Assert.Equal(expected, composedRequestUrl);
+        }
+
```

Comment:
I think it would be great to add a test that validates the final expected request url when we have the Query options (filter and orderby)present as well. 
The current tests just validate that we don't throw an error when we have them

Reply:
Could you also add a test that validates the concatenation of comma separated parameters?

Reply:
Good idea. Done.